File: client/src/app/notes/note.ts
export interface Note {
  _id: string;
  body: string;
  reuse: boolean;
  draft: boolean;
  toDelete: boolean;
}

export type reusable = 'true' | 'false';
export type draft = 'true' | 'false';
export type toDelete = 'true' | 'false';

File: server/src/main/java/umm3601/note/Note.java
package umm3601.note;

import org.mongojack.Id;
import org.mongojack.ObjectId;

public class Note {
  @ObjectId @Id
  public String _id;
  //public String owner;
  public String body;
  public boolean reuse;
  public boolean draft;
  public boolean toDelete;
}
